allennlp/tests/data/dataset_readers/text_classification_json_test.py

```diff
@@ -0,0 +1,46 @@
+# pylint: disable=no-self-use,invalid-name
+import pytest
+
+from allennlp.data.dataset_readers import TextClassificationJSONReader
+from allennlp.common.util import ensure_list
+from allennlp.common.testing import AllenNlpTestCase
+
+
+class TestTextClassificationJSONReader():
+
+    @pytest.mark.parametrize("lazy", (True, False))
+    def test_read_from_file_ag(self, lazy):
```

Comment:
what does `ag` mean?

Reply:
oh i see, it's a dataset.

Reply:
yeah, it's a sample of the ag-news corpus. i'll make that clear.